synapseclient/models/services/storable_entity.py

```diff
@@ -0,0 +1,103 @@
+"""Script used to store an entity to Synapse."""
+
+from typing import TYPE_CHECKING, Dict, Optional, Union
+
+from opentelemetry import trace
+
+from synapseclient import Synapse
+from synapseclient.api import (
+    post_entity,
+    put_entity,
+    create_access_requirements_if_none,
+)
+from synapseclient.core.utils import get_properties
+
+if TYPE_CHECKING:
+    from synapseclient.models import File, Folder, Project
+
+tracer = trace.get_tracer("synapseclient")
+
+
+async def store_entity(
```

Comment:
I intend for this to be replaced when https://sagebionetworks.jira.com/browse/SYNPY-1437 - can be worked on (Currently blocked by a few PLFM tickets)